qiskit/algorithms/gradients/finite_diff_estimator_gradient.py

```diff
@@ -28,29 +29,50 @@
 from .base_estimator_gradient import BaseEstimatorGradient
 from .estimator_gradient_result import EstimatorGradientResult
 
+if sys.version_info >= (3, 8):
+    # pylint: disable=no-name-in-module, ungrouped-imports
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
 
 class FiniteDiffEstimatorGradient(BaseEstimatorGradient):
     """
     Compute the gradients of the expectation values by finite difference method.
     """
 
-    def __init__(self, estimator: BaseEstimator, epsilon: float, options: Options | None = None):
+    def __init__(
+        self,
+        estimator: BaseEstimator,
+        epsilon: float,
+        method: Literal["central", "forward", "backward"] = "central",
```

Comment:
Since the gradients have been released already, this new argument needs to go last, otherwise using the arguments positionally 
```
FiniteDiffEstimatorGradient(my_estimator, my_epsilon, my_options)
```
wouldn't work the same as before anymore 🙂

Reply:
Sure... it's just unnatural. Maybe we should just make it a keyword-only argument?

Reply:
From my understanding `options` is only what affects the circuit evaluation since this is simply forwarded to the underlying primitives. Everything else (including all gradient settings) doesn't go there, which I think is good because then we can clearly document what the effect is.

We could make it keyword only, yes, since we started using that in the algorithms anyways it might be the most natural 🙂

Reply:
If the options is limited to being values that are passed to underlying primitive run options then its clear.  Given these are primitive-like and with some primitives themselves I am sure I have seen some with options more than run options supports I  just wanted to check. Having such as explicit parameters seems better to me, it exposes things more directly. It does mean presently though to change method (or epsilon) you need to create a new gradient object as there is no public way to update them.

Reply:
I think we can change the implementation to make it gradient's own run option. It is not necessary to transfer all values to primitives. 
(I agree the instance option is fine in this case, but if the user needs the run option...)